docs/source/sabre_transaction_family.rst

```diff
@@ -386,7 +386,7 @@ Create a contract registry with no version.
     repeated string owners = 2;
   }
 
-If the contract registry for the provided contract name already exists, the
+If the contract registry for the provided contract name already exists, then
```

Comment:
This change is not necessary.  The previous sentence is correct.

Reply:
There is  a double "the"

Reply:
Oops. Yes, you're right and I'm wrong.  (I was skimming too fast.) Sorry.

Also, thanks, @rranjan3, for fixing this and other errors!